{"name": "Проверка получения информации по некорректному ID", "status": "broken", "statusDetails": {"message": "TypeError: PetApi.__init__() missing 1 required positional argument: 'logger'", "trace": "self = <tests.test_pet_api.TestPetApi object at 0x10289f940>, id = '111111111111111111111'\n\n    @allure.title('Проверка получения информации по некорректному ID')\n    @pytest.mark.parametrize('id', ['922337203685477580e7', '0000000', '111111111111111111111'])\n    def test_get_pet_by_non_existent_id(self, id):\n>       pet_api = PetApi()\nE       TypeError: PetApi.__init__() missing 1 required positional argument: 'logger'\n\nid         = '111111111111111111111'\nself       = <tests.test_pet_api.TestPetApi object at 0x10289f940>\n\ntests/test_pet_api.py:53: TypeError"}, "parameters": [{"name": "id", "value": "'111111111111111111111'"}], "start": 1714327507454, "stop": 1714327507454, "uuid": "1ffcef0d-1aca-47ad-b31b-99a04da9819c", "historyId": "34083f21ceccdf1e90c1c1d10eb64d1a", "testCaseId": "acf0b1f06f555e3a99915b0d677338d3", "fullName": "tests.test_pet_api.TestPetApi#test_get_pet_by_non_existent_id", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_pet_api"}, {"name": "subSuite", "value": "TestPetApi"}, {"name": "host", "value": "d45763e349a4"}, {"name": "thread", "value": "97822-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_pet_api"}]}